pkg/apis/greenhouse/v1alpha1/team_types.go

```diff
@@ -47,3 +47,9 @@ type TeamList struct {
 func init() {
 	SchemeBuilder.Register(&Team{}, &TeamList{})
 }
+
+func (o *Team) GetConditions() StatusConditions {
+	return StatusConditions{} // OrganizationStatus is an empty struct so we don't need to get anything
```

Comment:
The lifecycle pkg will set default `Delete` condition on the resource it is reconciling (Team)

We need this for `EnsureDeleted` otherwise there will be an endless reconcile loop and finalizers will not be removed

Reply:
see here

https://github.com/cloudoperators/greenhouse/blob/6c50e675e6113fa0a18dbeafb6920890fc8f475e/pkg/lifecycle/reconcile.go#L87-L93

https://github.com/cloudoperators/greenhouse/blob/6c50e675e6113fa0a18dbeafb6920890fc8f475e/pkg/lifecycle/reconcile.go#L107-L114

Reply:
Done